.gitignore

```diff
@@ -24,3 +24,6 @@ node_modules
 
 # Daily output
 /daily_output.json
+
+distribution/*
+!distribution/.gitkeep
```

Comment:
Or... We could create the folder if not present when running the script. BTW does the ! sign make it required?

Reply:
No.. it means to not ignore that unique file.